frontend/Readme.md

```diff
@@ -39,7 +39,9 @@ bower install
 
 cd ..
 
-go run main.go
+go build
+
+./frontend.exe
```

Comment:
Is there any problem when you run 'go run main.go'? And this change does not work in other platform

Reply:
Hi, I've tried `go run main.go` but failed with error: "./main.go:50:2: undefined: Serve"
Since `Serve` function is defined in server.go, if we want to run go files directly, explicitly specify server.go is needed:
```bash
go run server.go main.go
```

Reply:
@crystaldust I think server.go should be moved out from main package

Reply:
@little-cui  go run does not work well for multi-file project and it should not be used anyway. @xiaoyimei please fix the PR by provide 2 separate command for running the frontend. frontend.exe for windows and ./frontend for linux.

Reply:
@yangbor i do not mean to run go 'run' command with multi-files, i think create a new package 'server' and move the server.go into it would be a better solution.

Reply:
@little-cui @xiaoyimei I suggest to put this command in a start.sh/start.bat script and handle the building and  running frontend in that and give this command of running the script in Readme